pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -209,6 +209,10 @@ func (s *Server) RunCA(grpc *grpc.Server, ca caserver.CertificateAuthority, opts
 		}
 	}
 
+	// Allow authorization with a previously issued certificate, for VMs
```

Comment:
I think it is not for authorization, but just for signing cert with identity

Reply:
The authenticator will check the caller certificate, against the current CA key, and extract the SAN - that's what we've been doing for mesh expansion.

"Authorization" is indeed not very accurate - there is no explicit authorization check, just 
authentication and implicit 'if workload already has a signed cert, allow issuing a fresh one with same SAN'.

AFAIK this is the same thing Citadel was doing, when mesh expansion was enabled.